pdf_craft/epub/toc_collection.py

```
@@ -75,3 +85,20 @@ def _find_or_append_toc_item(self, id: int, toc_items: list[TocItem]) -> TocItem
             self._id_to_toc_item[id] = toc_item
         return toc_item
+
+    def _clean_no_content_items(self, toc_items: list[TocItem]):
+        index = 0
+        while index < len(toc_items):
+            should_keep = True
+            toc_item = toc_items[index]
+            self._clean_no_content_items(toc_item.children)
+
+            if id(toc_item) not in self._having_body_toc_set and toc_item.children:
+                # 有子节点同时自己也没有内容，变为仅存在于目录中的章节有助于更好的阅读体验
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)